src/dune

```diff
@@ -12,7 +12,7 @@ Jbuild_plugin.V1.send
 (rule
  (targets Version.ml)
  (deps (universe))
- (action (run sh %%{dep:../tools/gen_version.sh} Version.ml))
+ (action (bash "%%{dep:../tools/gen_version.sh} Version.ml"))
```

Comment:
IIUC, `run` is necessary to prevent dune from using `system`, and unquoted `%%{` arg are necessary for dune to track dependencies correctly.

Reply:
This assumes bash is installed, can we just merge #471 given the diff is really small and tests have been run?